src/CPULSSolver.h

```diff
@@ -76,13 +76,192 @@ class CPULSSolver : public CPUSolver {
   void stabilizeFlux();
   void checkLimitXS(int iteration);
 
-  void tallyLSScalarFlux(segment* curr_segment, int azim_index, int polar_index,
-                         float* track_flux, FP_PRECISION direction[3]);
-
   FP_PRECISION getFluxByCoords(LocalCoords* coords, int group);
   void initializeLinearSourceConstants();
   double* getLinearExpansionCoeffsBuffer();
   FP_PRECISION* getSourceConstantsBuffer();
+
+/**
```

Comment:
Does this inline gain efficiency? And how about the executive file size increase?

Reply:
Turns out not. 
I am still testing a few things, which is why I keep force-pushing that branch